### PR TITLE
[webapi] Add RealSense tests

### PR DESCRIPTION
**Pull request by chunywang**
- Add tests for Photo, PhotoCapture API
- Add tests for new API: getInternalCameraIntrinsics,
  getVolumePreview

Impacted TCs num(approved): New 20, Update 0, Delete 0
Unit test Platform: Crosswalk Project for Windows 19.48.498.0
Unit test result summary: Pass 20, Fail 0, Blocked 0

BUG=https://crosswalk-project.org/jira/browse/XWALK-6345
BUG=https://crosswalk-project.org/jira/browse/XWALK-6545

### CODE REVIEW

File: webapi/webapi-realsense-xwalk-tests/realsense/support/adapter.js [hunk 1]
```diff
@@ -0,0 +1,556 @@
+/*
+ *  Copyright (c) 2014 The WebRTC project authors. All Rights Reserved.
```
Comment (zqzhang) [inline webapi/webapi-realsense-xwalk-tests/realsense/support/adapter.js]: Where do you get this source?

Comment (chunywang) [inline webapi/webapi-realsense-xwalk-tests/realsense/support/adapter.js]: from Realsense sample

**Comment by Honry** [comment]
Please reference the task id in your commit msg.

**Comment by zqzhang** [comment]
Will have an offline talk to improve these tests.

**Comment by chunywang** [comment]
Tomorrow morning, it's ok?

**Pull request merged**